.github/ISSUE_TEMPLATE/ui-copy-request.md

```diff
@@ -0,0 +1,38 @@
+---
+name: UI Copy Request
+about: Describe the UI text that you need us to work on.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Description 
+[Summarize your request here. What is the work, what is its current status, and how can we help? Feel free to include screenshots.]
```

Comment:
Would you mind elaborating on what you mean by "current status"? Are we asking what stage the copy's in? 

On a related note, are we offering to help design write copy that doesn't exist? Or, are we only committing to reviewing existing copy? If it's the former, should we include that detail here?